src/server.js

```diff
@@ -1,8 +1,8 @@
-const express = require("express");
```

Comment:
I think we should stop using auto-formatting....
Otherwise we will keep changing the whole code every time we take turns to edit the same files

Reply:
we should have a linter that can autoformat for us.